packages/scaffolder-cli/cli-e2e-test.config.js

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
i'd probably encourage moving `cli-e2e-tests` folder to something like `tests/e2e` instead or something to keep it a little cleaner